server/routes/blog.py

```
@@ -199,6 +246,144 @@ def send_newsletter(self, app, new_blog):
+                ).start()
+
+            return restful_response(
+                status="success",
+                message="Blog updated successfully",
+                data=blog.to_dict(),
+                status_code=200,
+            )
+        except ValueError as e:
+            db.session.rollback()
+            return restful_response(status="error", message=str(e), status_code=400)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)